build.gradle

```diff
@@ -712,12 +717,14 @@ jlink {
         requires 'jakarta.xml.bind'
         requires 'org.apache.commons.lang3'
         requires 'org.apache.commons.text'
+        requires 'org.apache.commons.logging';
```

Comment:
Move up one line. Alphabetical ordering - the current order is not logical

Reply:
The order can be crucial just simple alphabetical  order will not work properly (there is a comment stating this)